enhancements/storage/vsphere-storage-check.md

```diff
@@ -0,0 +1,228 @@
+---
+title: vsphere-monitoring-operator
+authors:
+  - "@jsafrane"
+reviewers:
+  - "@hekumar"
+  - "@jspeed"
+  - "@jcpowermac"
+  - # someone from installer
+approvers:
+  - TBD
+creation-date: 2020-11-10
+last-updated: 2020-11-10
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# vSphere monitoring for OCP clusters
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Below we propose to check if OpenShift clusters installed into vSphere environment is / can be fully functional by introducing a new operator and periodical checks.
+
+## Motivation
+
+Our support frequently faces OCP clusters on vSphere that is either mis-configured.
```

Comment:
is there a missing part to this sentence?

> that is either mis-configured.

seems like there should be an "or ..."

Reply:
fixed